packages/react/src/Header/Header.tsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import clsx from 'clsx'
+import { ForwardedRef, forwardRef, HTMLAttributes, ReactNode } from 'react'
+import { Heading } from '../Heading'
+import { Logo, LogoBrand } from '../Logo'
+import { VisuallyHidden } from '../VisuallyHidden'
+
+export interface HeaderProps extends HTMLAttributes<HTMLElement> {
+  logoBrand?: LogoBrand
+  logoLink?: string
+  logoLinkTitle?: string
+  title?: string
+  menu?: ReactNode
+}
+
+export const Header = forwardRef(
+  (
+    { className, logoBrand, logoLink, logoLinkTitle, title, menu, ...restProps }: HeaderProps,
```

Comment:
Let's set defaults here, `logoBrand="amsterdam"`, `logoLinkTitle="Ga naar homepage"`. That way we could get rid of a bunch of the checks below

Reply:
‘Ga naar *de* homepage’?

Reply:
Why would you disallow people from having a header without a link? There can be cases like a landingpage (without any navigation) where the header is a static thing right?

Reply:
It's described [here](https://www.amsterdam.nl/digitalservices/design-system/atomen-moleculen/logo-and-favicon/#hab-7cac11-e64e-44cb-acb7-23dbfc88b972), a logo in a header links to the apps homepage. If the app doesn't have a homepage, it links to the page of origin on amsterdam.nl

Reply:
Ok